src/app/vertextool/qgsvertexeditor.cpp

```diff
@@ -457,6 +459,22 @@ void QgsVertexEditor::keyPressEvent( QKeyEvent *e )
     // Override default shortcut management in MapCanvas
     e->ignore();
   }
+  else if ( e->matches( QKeySequence::Copy ) )
+  {
+    if ( !mTableView->selectionModel()->hasSelection() )
+      return;
+    QString text;
+    QItemSelectionRange range = mTableView->selectionModel()->selection().first();
+    for ( int i = range.top(); i <= range.bottom(); ++i )
+    {
+      QStringList rowContents;
+      for ( int j = range.left(); j <= range.right(); ++j )
+        rowContents << mVertexModel->index( i, j ).data().toString();
+      text += rowContents.join( ',' );
```

Comment:
A tab separator would be more appropriate here, as the comma will be confusing in locales that use it as a decimal separator.